src/tests/integration/test-data/deployment-main.expected

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.8.13;
+
+import { Vm } from "forge-std/Vm.sol";
+
+import 'src/DeploymentSummary1Code.sol';
```

Comment:
```suggestion
import { DeploymentSummary1Code } from 'src/DeploymentSummary1Code.sol';
```
This should be as suggested to have a consistent style

Reply:
Fixed.